newrelic_lambda_cli/layers.py

```diff
@@ -115,7 +115,16 @@ def _add_new_relic(input, config, nr_license_key):
     if runtime_handler and handler != runtime_handler:
         update_kwargs["Environment"]["Variables"]["NEW_RELIC_LAMBDA_HANDLER"] = handler
 
-    if input.enable_extension:
+    if input.enable_extension and not utils.supports_lambda_extension(runtime):
+        warning(
+            "The %s runtime does not support Lambda Extensions, reverting to a "
+            "CloudWatch Logs based ingestion. Make sure you run `newrelic-lambda "
+            "integrations install` command to install the New Relic log ingestion "
+            "function and `newrelic-lambda subscriptions install` to create the log "
+            "subscription filter." % runtime
+        )
```

Comment:
Should we also include the function name/arn in the warning, for context in the case of bulk operations?